test/index.test.js

```diff
@@ -31,7 +31,7 @@ describe('instantiation', () => {
     );
   });
 
-  // TOOD: Warn instead of throw?
+  // TODO: Warn instead of throw?
```

Comment:
I'm sorry for this :sweat_smile:. Please don't be ashamed, I do much worse. I fixed it just because else code analysis doesn't pick it up and never warns that there is a TODO somewhere to work on. Everything is great!

Reply:
Oh I don't mind at all. I'm just amazed that I spent so much time looking at that code over the years and never spotted it! 😆